introduction.md

```diff
@@ -1,2 +1,6 @@
 Hi, wdhays2 here. Nice to meet you.
-http://tryruby.org/levels/1/challenges/0 is my favorite.
\ No newline at end of file
+http://tryruby.org/levels/1/challenges/0 is my favorite.
+My skillset includes:
+* html
+* css 
+* javascript.
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?